lib/rubocop/rspec/example.rb

```diff
@@ -4,7 +4,7 @@ module RuboCop
   module RSpec
     # Wrapper for RSpec examples
     class Example < Concept
-      def_node_matcher :extract_doc_string,     '(send _ _ $str ...)'
+      def_node_matcher :extract_doc_string,     '(send _ _ $_ ...)'
```

Comment:
I wonder if we should add a couple of examples to example_spec.rb

Reply:
@bquorning agree, fixed